book/website/collaboration/leadership/leadership-features.md

```diff
@@ -0,0 +1,102 @@
+(cl-leadership-features)=
+# Important Features of Leadership
+
+Leaders empower others to identify solutions and make their own decisions to address the challenges they face.
+[Monica Cox](https://en.wikipedia.org/wiki/Monica_Cox), an awardee of Purdue University's Faculty Award of Excellent for Leadership, once referred to leadership as "the highest way of activism" ([reference](https://twitter.com/DrMonicaCox/status/1299346141386022913)).
```

Comment:
```suggestion

```{cite}
"A leader is anyone who takes responsibility for finding the potential in people and processes and has the courage to develop that potential. Leadership is not about titles or the corner office. It's about the willingness to step up, put yourself out there, and lean into courage.”
- Brene Brown, [Dare to Lead](https://brenebrown.com/hubs/dare-to-lead/)
```